packages/api-explorer/i18n/en.json

```diff
@@ -50,5 +50,17 @@
   "auth.oauth2.info": "Authenticate via OAuth2",
   "auth.oauth2.authorization": "Authorization",
   "auth.oauth2.bearer": "Bearer",
+  "button_delete_all": "All",
```

Comment:
Place these below `authForm.input.placeholder` separated with a space. (do the same with the italian)

Also check if these are actually used by jsoneditor, if not delete them

Reply:
removed